src/System.Net.Http/ref/project.json

```diff
@@ -3,14 +3,15 @@
     "System.Runtime": "4.0.0",
     "System.Threading.Tasks": "4.0.0",
     "System.IO": "4.0.0",
-    "System.Net.Primitives": "4.0.0",
+    "System.Net.Primitives": "4.0.10",
+    "System.Security.Cryptography.X509Certificates": "4.1.0-rc3-24008-00",
```

Comment:
Does that mean I should add these other assemblies to NETStandard.Library targets file:

* System.Security.Cryptography.Algorithms
* System.Security.Cryptography.Encoding
* System.Security.Cryptography.Primitives